application/dash_application/table_builder.py

```diff
@@ -77,9 +80,14 @@ def cutoff_table(table_id: str, data: DataFrame, limits: List[Tuple[str, str, fl
             },
             *({
                 "if": {"column_id": name, "filter_query": "{%s} contains 'Failed'" % name},
-                "backgroundColor": "pink"
+                "backgroundColor": "mistyrose"
+
+            } for (name, _, _) in limits),
+            *({
+                "if": {"column_id": name, "filter_query": "{%s} = 'Missing'" % name},
+                "backgroundColor": "mintcream"
```

Comment:
Can this colour be changed? It's the same saturation and intensity as the grey used for alternating rows (see attached) for nearly all forms of colourblindness.
![redblind](https://user-images.githubusercontent.com/8324996/70638519-0906a280-1c07-11ea-9463-6fdac582889e.png)

Reply:
Is now papayawhip